testing/integration/integration_util.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/google/exposure-notifications-server/internal/database"
+	"github.com/google/exposure-notifications-server/internal/monolith"
+)
+
+func StartSystemUnderTest(tb testing.TB, ctx context.Context) *monolith.MonoConfig {
+	tb.Helper()
+
+	if testing.Short() {
+		tb.Skipf("🚧 Skipping integration tests (short!")
+	}
+
+	if skip, _ := strconv.ParseBool(os.Getenv("SKIP_INTEGRATION_TESTS")); skip {
+		tb.Skipf("🚧 Skipping integration tests (SKIP_INTEGRATION_TESTS is set)!")
+	}
+
+	database.NewTestDatabase(tb)
+
+	monoConfig, err := monolith.RunServer(ctx)
```

Comment:
How will test know about the url/port to use for connection to monolith?

Reply:
The monoConfig should contain that info.  Definition is here: https://github.com/google/exposure-notifications-server/blob/14f2d3aaa1103466f4ff252ae076197d90d37f5f/internal/monolith/monolith_util.go#L45-L55

Reply:
This will stand up the Monolith server and it will be torn down at the end.  No piping of addresses back and forth yet.  I need to separate the RunServer method into two different methods, one that generates the config without starting anything, and a second that runs.